src/proj.h

```diff
@@ -710,6 +710,9 @@ typedef enum
     PJ_TYPE_VERTICAL_REFERENCE_FRAME,
     PJ_TYPE_DYNAMIC_VERTICAL_REFERENCE_FRAME,
     PJ_TYPE_DATUM_ENSEMBLE,
+    PJ_TYPE_TEMPORAL_DATUM,
```

Comment:
those new values must be added at the end of the enumerations, otherwise this changes the ABI